### PR TITLE
feat: Added version command & created readme

### LINKED ISSUE #6: A `README` for CLI

### PR DESCRIPTION
**Pull request by Murtatrxx**
~~Also it includes some changes to logic.~~

Closes #6

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Sern CLI
+
+Our CLI is allowing you to create & streamline your Discord bot project without writing a single line of code!
+
+😁 **User Friendly Usage** <br>
```
Comment (jacoobes) [inline README.md:5]: **User Friendly Usage** -> **User Friendly**

File: README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+🔥 **At blazing Speeds** <br>
```
Comment (jacoobes) [inline README.md:6]: Remove this blazing speeds thing, cli doesnt need to be blazing fast

File: README.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+🌱 **Efficiently** <br>
```
Comment (jacoobes) [inline README.md:7]: add : Batteries included cli, 
efficiently -> efficient

File: README.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+```sh
+yarn add @sern/cli@latest
+```
+
+## Getting Started
+
+When you install the CLI, you can use our commands with **sern** prefix. Usage is simple;
+
+`sern <command> (opt)<flag>`
```
Comment (jacoobes) [inline README.md:23]: add code block maybe?
```sh
sern <command> (opt)<flag>
```

**Review by jacoobes — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (Murtatrxx) [reply README.md:23]: This is not an actual command, it's only a usage. Btw other things you requested will be here asap

File: README.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+npm install @sern/cli@latest
+```
+
+```sh
+yarn add @sern/cli@latest
+```
+
+## Getting Started
+
+When you install the CLI, you can use our commands with **sern** prefix. Usage is simple;
```
Comment (jacoobes) [inline README.md:21]: end of line : use a colon ( : ) instead of semicolon ( ;)

File: README.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
+# Sern CLI
+
+Our CLI is allowing you to create & streamline your Discord bot project without writing a single line of code!
```
Comment (jacoobes) [inline README.md:3]: Our CLI allows you to setup and manage Discord bot projects without writing a line of code!

**Review by jacoobes — CHANGES_REQUESTED** [review]

File: src/commands/version.js [hunk 7]
```diff
@@ -0,0 +1,3 @@
+export default function version() {
+    console.log('SernHandler CLI v0.1.1');
```
Comment (EvolutionX-10) [inline src/commands/version.js:2]: use package.json to get version instead of hard coding it

File: src/index.js [hunk 8]
```diff
@@ -1,7 +1,8 @@
 #!/usr/bin/env node
 
-import { init } from './commands/init.js';
-import { help } from './commands/help.js';
+import init from './commands/init.js';
+import help from './commands/help.js';
+import version from './commands/version.js';
```
Comment (EvolutionX-10) [inline src/index.js:5]:
```suggestion
import { init } from './commands/init.js';
import { help } from './commands/help.js';
import { version } from './commands/version.js';
```

File: src/index.js [hunk 9]
```diff
@@ -19,6 +20,8 @@ const commands = new Map([
 	['help', help],
 	['', help],
 	['init', init],
+        ['version', version],
```
Comment (EvolutionX-10) [inline src/index.js:23]: been thinking, how about we tell the version in help itself
@jacoobes and @Allyedge any opinions?

**Review by EvolutionX-10 — CHANGES_REQUESTED** [review]

File: src/commands/init.js [hunk 10]
```diff
@@ -16,7 +16,7 @@ import { cloneRepo, installDeps } from '../utilities/install.js';
 import { editDirs, editMain } from '../utilities/edits.js';
 const { prompt } = prompts;
 
-export async function init({ flags }) {
+export default async function init({ flags }) {
```
[the remark this answers is not in the document]
Comment (Murtatrxx) [reply src/commands/init.js:19]: Too late

File: src/commands/help.js [hunk 11]
```diff
@@ -1,3 +1,3 @@
-export function help() {
+export default function help() {
```
[the remark this answers is not in the document]
Comment (Murtatrxx) [reply src/commands/help.js:1]: Too late

File: src/index.js (hunk 8 above)
Comment (Murtatrxx) [reply src/index.js:5]: I used default exports

File: src/index.js (hunk 9 above)
Comment (Murtatrxx) [reply src/index.js:23]: A separate command is needed, but we can include version with help command

File: src/commands/version.js (hunk 7 above)
Comment (Murtatrxx) [reply src/commands/version.js:2]: I’m mazy for that

File: README.md [hunk 12]
```diff
@@ -0,0 +1,27 @@
...
+🌱 **Efficiently** <br>
+
+## Installation
+
+```sh
+npm install @sern/cli@latest
+```
+
+```sh
+yarn add @sern/cli@latest
```
[the remark this answers is not in the document]
Comment (Murtatrxx) [reply README.md]: Done

File: README.md [hunk 13]
```diff
@@ -0,0 +1,27 @@
...
+Our CLI is allowing you to create & streamline your Discord bot project without writing a single line of code!
+
+😁 **User Friendly Usage** <br>
+🔥 **At blazing Speeds** <br>
+🌱 **Efficiently** <br>
+
+## Installation
+
+```sh
+npm install @sern/cli@latest
```
[the remark this answers is not in the document]
Comment (Murtatrxx) [reply README.md]: Done

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (Murtatrxx) [reply README.md]: Completely changed these

File: README.md (hunk 6 above)
Comment (Murtatrxx) [reply README.md]: Ok

File: README.md (hunk 5 above)
Comment (Murtatrxx) [reply README.md]: Removed it

File: README.md (hunk 2 above)
Comment (Murtatrxx) [reply README.md]: Ok

File: src/commands/version.js (hunk 7 above)
Comment (EvolutionX-10) [reply src/commands/version.js:2]: no

File: src/index.js (hunk 8 above)
Comment (EvolutionX-10) [reply src/index.js:5]: remove em, they have issues

File: src/index.js (hunk 9 above)
Comment (EvolutionX-10) [reply src/index.js:23]: yeah do it

File: README.md [hunk 14]
```diff
@@ -0,0 +1,33 @@
...
+
+Our CLI allows you to setup and manage Discord bot project without writing a single line of code!
+
+😁 **User Friendly** <br>
+💦 **Easy-to-use** <br>
+🌱 **Efficient** <br>
+💪 **Powerful** <br>
+
+## Installation
+
```
Comment (Allyedge) [inline README.md:11]: You could add `NPM`, `Yarn` and `PNPM` titles for the installation part. Would make it a bit more user friendly I believe.

**Review by Allyedge — CHANGES_REQUESTED** [review]
Looks good. Though I believe titles (or any kind of large text) for package manager names would be great in `README.md`.

File: src/commands/version.js [hunk 15]
```diff
@@ -0,0 +1,7 @@
+import { createRequire } from 'module';
+const require = createRequire(import.meta.url);
+const { version } = require('../../package.json');
+
+export function version() {
+    console.log('SernHandler CLI v' + version);
+}
```
Comment (EvolutionX-10) [inline src/commands/version.js:7]:
```suggestion
const { version: v } = require('../../package.json');

export function version() {
    console.log(`SernHandler CLI v${v}`);
}
```
js being js

**Review by EvolutionX-10 — CHANGES_REQUESTED** [review]

**Review by EvolutionX-10 — DISMISSED** [review]

File: README.md [hunk 16]
```diff
@@ -0,0 +1,33 @@
+# Sern CLI
+
+Our CLI allows you to setup and manage Discord bot project without writing a single line of code!
```
Comment (jacoobes) [inline README.md:3]: and manage Discord bot projects <- u forgot s

File: README.md [hunk 17]
```diff
@@ -0,0 +1,33 @@
...
+
+😁 **User Friendly** <br>
+💦 **Easy-to-use** <br>
```
Comment (jacoobes) [inline README.md:6]: Why not just '💦 **Simple**' instead of 💦 **Easy-to-use**

**Review by jacoobes — CHANGES_REQUESTED** [review]

**Comment by EvolutionX-10** [comment]
Updated version #25

**Pull request closed without merging**